### DISCUSSION TITLE
Nation Building and Society Simulation

### DISCUSSION THREAD

**Discussion by mokun** [mars-sim/mars-sim#734] [Ideas]
Currently, mars-sim focuses on the simulation of individual settlers' actions and details simulation of building functions for a building and how these actions and functions may affect the settlement as a whole. 

In micromanaging what settlers do in every frame, it lacks the big picture on what the settlement will become.

We can imagine that with more and more settlements become both self-sufficient and interdependent, it is difficult for a player to keep track of why each malfunction happens and how each mission is doing.

In light of that, we could change some underlining core mechanics to make mars-sim a future macro-management simulator. 

By creating a new game mode called "Society Mode", we can simulate macroscopically how human societies and tribes (simulating each human group, instead of simulating each settler) will evolve as Mars itself is being developed into a cluster of settlements and eventually into a nation state. 

In this mode, we will focus on society-building or nation-building. 

As these tribes/nations flourish, they will take up land and produce more economically and spread out on Mars.

They will establish government, make pacts and agreements with others, take their stance politically and form their own national and cultural identity.

Of course, it will also be fascinating to simulate how these new emerging nation-states on Mars will interact with Moon colonies and Earth governments.

Feel free to share your idea along this line of thought in this thread.

**Comment by bevans2000** [comment]
What would be different in this "society mode" ?

**Reply by mokun** [reply]
> What would be different in this "society mode" ?

I see that in this mode, the player commander or the mayor of a settlement would be given the role of deciding on higher level of things on behalf of the whole settlement, instead of overseeing the actions of the individuals.
